### PR TITLE
facets: adds facets for other collections

### PR DESCRIPTION
**Pull request by ioannistsanaktsidis**
Signed-off-by: Ioannis Tsanaktsidis <ioannis.tsanaktsidis@cern.ch>

### CODE REVIEW

**Comment by ioannistsanaktsidis** [comment]
I open this PR so as to keep track of the progress of adding facets to all collections.

**Comment by ioannistsanaktsidis** [comment]
@jmartinm  The document type is added in the hep collection. Seems to work fine from the first test.

**Comment by ioannistsanaktsidis** [comment]
@jmartinm I fixed the design issues so as to be better for the User testing. You can test and merge the PR. See you tomorrow.Cheers.

File: Procfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 web: inveniomanage runserver
-cache: redis-server
+cache: redis-server /usr/local/etc/redis.conf
```
Comment (jmartinm) [inline Procfile]: this shouldnt be merged

File: install.sh [hunk 2]
```diff
@@ -13,6 +13,8 @@ pip install unittest2
 # Install INSPIRE
 pip install -r requirements.txt
 
+pip freeze
```
Comment (jmartinm) [inline install.sh]: this was part of your testing I guess

**Comment by jmartinm** [comment]
Please, fix the PEP8 failures in Travis

**Comment by jmartinm** [comment]
> @jmartinm I fixed the design issues so as to be better for the User testing. You can test and merge the PR. See you tomorrow.Cheers.

Which ones are addressed in this PR?

**Comment by ioannistsanaktsidis** [comment]
@jmartinm I don't think there is an open issue for this one.

File: inspirehep/config.py [hunk 3]
```diff
@@ -293,14 +293,18 @@
     "author": {
         'a': ["authors.full_name"],
         'p': ["authors.full_name"],
-        'e': ['exactauthor.raw'],
+        'e': ['exactauthor.raw']
```
Comment (jmartinm) [inline inspirehep/config.py]: why the removal of the comma?

File: inspirehep/modules/records/receivers.py [hunk 4]
```diff
@@ -35,3 +34,44 @@ def populate_inspire_subjects(recid, json):
...
+                    break
+                elif element['primary'].lower() == 'proceedings':
+                    inspire_doc_type.append(element['primary'].lower())
+                    break
+                elif element['primary'].lower() == 'conferencepaper':
+                    inspire_doc_type.append('conference paper')
+                    break
+                elif json.get('publication_info', []):
+                    for field in json.get('publication_info', []):
+                        if 'page_artid' in field:
```
Comment (jmartinm) [inline inspirehep/modules/records/receivers.py]: Here it is the opposite. It is a preprint if there is NO `page_artid`

File: install.sh [hunk 5]
```diff
@@ -42,7 +42,7 @@ inveniomanage database init --user=root --password= --yes-i-know || echo ':('
 inveniomanage database create --quiet || echo ':('
 
 # Start Celery
-celery worker -E -A invenio_celery.celery --workdir=$VIRTUAL_ENV 1> /dev/null &
+celery worker -E -A invenio_celery.celery --loglevel=INFO --workdir=$VIRTUAL_ENV &
```
Comment (jmartinm) [inline install.sh]: remove this

**Comment by jmartinm** [comment]
In Jobs there are 2 facets with same title

screen shot 2015-12-01 at 2 29 57 pm

**Comment by jmartinm** [comment]
Institutions facet by country gives no results when clicking on a facet?

**Comment by ioannistsanaktsidis** [comment]
With the latest update this should be fixed.

**Comment by ioannistsanaktsidis** [comment]
This was happening before the latest amend. In the last one I did these
problems are fixed. Maybe you have one version before the last?

**Comment by jmartinm** [comment]
Also ICN was not working.

screen shot 2015-12-01 at 2 32 16 pm

I am not sure the title ICN is understandable though.

**Comment by jmartinm** [comment]
The version I have is the one from this morning deployed in https://qa.inspirehep.net/search

**Comment by ioannistsanaktsidis** [comment]
Then yes this is the old version, after the changes most of the problems
are handled now. Any proposals about changing ICN? Sth like Filter by Lab?

2015-12-01 14:33 GMT+01:00 Javier Martin Montull <notifications@github.com>:

**Comment by jmartinm** [comment]
@annetteholtkamp what do you think about this `Filter by ICN` ? See https://qa.inspirehep.net/search?cc=experiments

Is there a more human friendly name?

**Comment by ioannistsanaktsidis** [comment]
@jmartinm i have updated the PR with the latest changes. I think after you test it we can merge it and continue any new staff from that point. Cheers.

**Comment by jmartinm** [comment]
Merged

33acf0e facets: improvements
82c4eca facets: adds facets for other collections

**Pull request closed without merging**